package.json

```diff
@@ -5,10 +5,11 @@
   "main": "server.js",
   "scripts": {
     "build-css": "./css-build",
+    "coverage-clear": "rm -Rf coverage",
```

Comment:
How does this affect the CI as it will be run there as well and the `coverage` directory is not checked in?